src/accountsdb/accounts_file.zig

```diff
@@ -259,7 +267,8 @@ pub const AccountInFile = struct {
 /// Analogous to [AccountStorageEntry](https://github.com/anza-xyz/agave/blob/4c921ca276bbd5997f809dec1dd3937fb06463cc/accounts-db/src/accounts_db.rs#L1069)
 pub const AccountFile = struct {
     // file contents
-    memory: []align(std.mem.page_size) u8,
+    file: std.fs.File,
```

Comment:
One concern I have here is that this moves us back from lots of mmap'd data to potentially lots of open file handles, which may wind up with us running into the "fd quota exceeded" errors again; are there any specific mitigations you implemented? If no, that's fine, we can work around them by increasing the fd quota like before, however in future we ought to get around to replacing stored file handles with some userland reference that we can use to derive the file handle on demand (ie the file id + slot pair used to derive the file name).

Non-actionable comment, feel free to mark as resolved if you don't have anything to comment and didn't implement any specific mitigations that I missed.

Reply:
> which may wind up with us running into the "fd quota exceeded" errors again

Hm, I guess we'll have to wait and see. Hopefully not.

> we ought to get around to replacing stored file handles with some userland reference that we can use to derive the file handle on demand (ie the file id + slot pair used to derive the file name).

Sounds easily doable, makes sense to me